pages/common/view.md

```diff
@@ -2,6 +2,7 @@
 
 > A read-only version of `vim`.
 > This is equivalent to `vim -R`.
+> More information: <https://www.vim.org>.
```

Comment:
```suggestion
> More information: <https://www.vim.org/docs>.
```

Reply:
```suggestion
> More information: <https://manned.org/view>.
```

Reply:
I've read somewhere about favoring the authors' information pages over others. Isn't this the case @navarroaxel?

Reply:
By the way, thank you all for being so supportive! (:

Reply:
Yeah, our first option it's the author's website, else `manned.org` is the first option.

Reply:
So which suggestion should i commit? 
I am confused because to me https://vim.org is the author's website😭

Reply:
Not really, In Arch Linux `view` is part of the traditional `vi` package, it isn't installed with `vim`.
In Ubuntu, and Fedora: `view` it's just an alias for the same `vim`, and the package `vi` doesn't exists.
Ok, leave as-is.

Reply:
I am in favor of keeping `vim.org`